README.md

```diff
@@ -1 +1,88 @@
-# angular-concepts-readme
+# Concepts
+
+## Overview
+
+There are some core concepts that every Angular developer will need to familiarise themselves with. This will teach you the core concepts that you will be using in every Angular application you make.
+
+## Objectives
+
+- Define MVC, MVVM
+- Describe two-way data-binding
+- Demonstrate updating Models and Views without two-way binding
+- Describe Angular's two-way binding with flow charts
+- Describe Dependency Injection
+- Describe Angular's Dependency Injection
+- Describe a Single Page Application and common features
+- Demonstrate server-side rendering concepts
+- Describe Single Page Applications with Angular
+
+## MVC/MVVM
+
+You might have seen the term MVC/MVVM being thrown around. These stand for Model-View-Controller/Model-View-ViewModel respectively. These is how you architecture your software. They are both quite similar, however there are a few differences -
+
+### MVC (Model-View-Controller)
+
+Three parts to MVC; model, view, controller. The model manages the fundamental behaviours and data of the application. This could be retrieved from a database or any other sort of data store.
+
+The view is the user interface. This takes data from the model and displays it nicely for the user. Any user interaction in the view is then handed over to the controller.
+
+The controller receives user input and makes calls to the model and view to update/modify either. This keeps the main aspects of our application separate.
+
+### MVVM (Model-View-ViewModel)
```

Comment:
You don't mentioned the MV of MVVM in this section - this becomes confusing later when you start referencing models. Are models in Angular the same as models in Rails? When you say 'model', are you referring to the 'model' of a Rails app? Be accessible but clear: "In a Rails application, our models represent data in our database. In Angular..."